# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/tree_util/IntermNode_util.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/compiler/translator/tree_util/IntermNode_util.h
--- a/gfx/angle/checkout/src/compiler/translator/tree_util/IntermNode_util.h	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/translator/tree_util/IntermNode_util.h	Fri Nov 08 17:06:00 2019 +0000
@@ -10,6 +10,7 @@
 #define COMPILER_TRANSLATOR_INTERMNODEUTIL_H_
 
 #include "compiler/translator/IntermNode.h"
+#include "compiler/translator/tree_util/FindFunction.h"
 
 namespace sh
 {
@@ -42,6 +43,13 @@
                                TIntermTyped *initializer,
                                TQualifier qualifier,
                                TIntermDeclaration **declarationOut);
+const TVariable *DeclareInterfaceBlock(TIntermBlock *root,
+                                       TSymbolTable *symbolTable,
+                                       TFieldList *fieldList,
+                                       TQualifier qualifier,
+                                       const TMemoryQualifier &memoryQualifier,
+                                       const char *blockTypeName,
+                                       const char *blockVariableName);
 
 // If the input node is nullptr, return nullptr.
 // If the input node is a block node, return it.
@@ -65,4 +73,4 @@
 
 }  // namespace sh
 
-#endif  // COMPILER_TRANSLATOR_INTERMNODEUTIL_H_
\ No newline at end of file
+#endif  // COMPILER_TRANSLATOR_INTERMNODEUTIL_H_